### PR TITLE
Add basic vote instruction decoder

### PR DESCRIPTION
**Pull request by ripatel-jump**
Adds basic ability to pretty print vote instructions, nothing more.

### CODE REVIEW

File: programs/vote/Vote.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+func (inst *Vote) EncodeToTree(parent treeout.Branches) {
+	parent.Child(format.Program(ProgramName, ProgramID)).
+		ParentFunc(func(programBranch treeout.Branches) {
+			programBranch.Child(format.Instruction("Vote")).
+				ParentFunc(func(instructionBranch treeout.Branches) {
+					// Parameters of the instruction:
+					instructionBranch.Child("Params").ParentFunc(func(paramsBranch treeout.Branches) {
+						paramsBranch.Child(format.Param("Slots", fmt.Sprintf("%v", inst.Slots)))
```
Comment (gagliardetto) [inline programs/vote/Vote.go:98]:
```suggestion
						paramsBranch.Child(format.Param("Slots", inst.Slots))
```

File: programs/vote/Vote.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+						paramsBranch.Child(format.Param("Hash", inst.Hash.String()))
```
Comment (gagliardetto) [inline programs/vote/Vote.go:99]:
```suggestion
						paramsBranch.Child(format.Param("Hash", inst.Hash))
```

File: programs/vote/Vote.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+						var ts time.Time
+						if inst.Timestamp != nil {
+							ts = time.Unix(*inst.Timestamp, 0).UTC()
```
Comment (gagliardetto) [inline programs/vote/Vote.go:102]: Is it in milliseconds?

You could also use [`solana.UnixTimeMilliseconds`](https://github.com/gagliardetto/solana-go/blob/475b9d64e462d5857898aaddd118e4bbe1359248/types.go#L36) as the type for it.
```suggestion
							ts = time.Unix(0, int64(*inst.Timestamp)*int64(time.Millisecond)).UTC()
```

Comment (ripatel-jump) [reply programs/vote/Vote.go:102]: No, seems to be in seconds

**Pull request merged**